node/sc/mainbridge_test.go

```diff
@@ -116,3 +117,79 @@ func TestMainBridge_basic(t *testing.T) {
 
 	//TODO more test
 }
+
+// TestMainBridge_removePeer tests correct removal of a peer from `MainBridge.peers`.
+func TestMainBridge_removePeer(t *testing.T) {
+	// Create a MainBridge (it may have 0 peers)
+	mBridge := testNewMainBridge(t)
+	defer mBridge.chainDB.Close()
+
+	// Prepare a bridgePeer to be added and removed
+	nodeID := "0x1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d1d"
```

Comment:
How about like this?
```
	key1, _ = crypto.GenerateKey()
	addr1 = crypto.PubkeyToAddress(key1.PublicKey)
	nodeID1 = discover.PubkeyID(&key1.PublicKey)
```

Reply:
Thank you for the suggestion.
Random nodeID may cause a test failure in line 138 `mBridge.removePeer("0x11111111")`(The possiblity is very low though).
So, I didn't change the code here, changed the same code in line 153.